### PR TITLE
Mind the code

### PR DESCRIPTION
**Pull request by KimberleyCook**
Mind the Code blogpost

### CODE REVIEW

File: source/_posts/2015-05-24-london-codebars.markdown [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+date: 2015-05-24 17:35:38 +0100
+comments: true
+author: Kimberley Cook
+author_email: kimberleycook91@gmail.com
+categories: london south-london codebar
+---
+
+The aim of this post is to clear up some confusion regarding the East London and South London codebar workshops. 
+
+As a coach or student, you have to sign up to the workshops seperately via the website. This means if you want to go on the East London workshop, sign up to this one. This means you will not receive invites for the South London workshop, and vice-versa. If you want to attend both workshops, you need to sign up to them seperately. This will mean you will start receiving invites for both London workshops. BUT, please do not then confirm attendance for both if they are on the same day. We do not have time to go through names and remove you from a list. So please bare this in mind when confirming your attendance for a workshop.
```
Comment (deniseyu) [inline source/_posts/2015-05-24-london-codebars.markdown]: slight typo - 'bear this in mind', not 'bare' :-)

File: source/_posts/2015-06-17-mind-the-code.markdown [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+date: 2015-06-17 22:17:50 +0100
+comments: true
+author: Kimberley Cook
+author_email: kimberleycook91@gmail.com
+categories: london mind-the-code codebar
+---
+
+On Saturday 4th June we organised Mind the Code, a day dedicated to mindfulness. Our aim was to run a slightly different event to anything we have ever ran before, where the day consisted of talks, workshops and yoga.
+
+The day began with a few lightning talks from all of our sponsors and the first main talk of the day by Emily Atkinson ([@emilyatk](https://twitter.com/emilyatk)), developer and product manager at ([Moo](http://uk.moo.com). Emily’s talk focused on coding mindfully. The first half of her talk focused on viewing code as a means to solving a problem. She discussed that as a developer you do not sit down to ‘write code’, you sit down to solve a problem. The second half of her talk was trying to get the audience to think about how other people view your code, and compared this to the foundations of a house. You want a solid base so that other developers can jump into the project and build upon your code without ruining your previous work.
```
Comment (Rosa-Fox) [inline source/_posts/2015-06-17-mind-the-code.markdown]: Need to add the closing bracket to (Moo)

File: source/_posts/2015-06-17-mind-the-code.markdown [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+[![Mind the code - Emily Atkinson](/images/MTC-EmilyAtkinson.jpg)]()
+
+The second part to the day was an exercise on domain modelling. Students and coaches were paired up randomly and asked to design a new system for managing the London Tube system. Using paper and pens, we asked them to think about the concepts behind this model, classes, and who would be responsible for these classes and the relationship between them. 
+
+[![Mind the Code - Exercise 1](/images/MTC-Exercise1.jpg)]()
+
+Once the first half of the day was completed, the first Yoga session was taken by Sandrine. This was then followed by lunch, provided by [Pronto](http://pronto.co.uk).
+
+This was then followed by the start of our main coding workshop. The task here was to pick one of our set ([projects](https://github.com/deniseyu/mind-the-code/tree/master/projects) and begin building it with their coaches. The projects included a yoga slot machine, yoga lullabies, yoga pose name generator, online mode ring and a to-do list. The aim of this workshop was to get the students started on a projects of their choice.
```
Comment (Rosa-Fox) [inline source/_posts/2015-06-17-mind-the-code.markdown]: Maybe change 'This was then followed' as is used in the line above to 'Next began our main coding workshop'

**Comment by matyikriszta** [comment]
thanks @KimberleyCook :+1:

**Pull request merged**